src/net/net.c

```diff
@@ -121,7 +121,7 @@ _Bool Net_StringToSockaddr(const char *s, struct sockaddr_in *saddr) {
 
 	struct addrinfo *info;
 	if (getaddrinfo(node, service, &hints, &info) == 0) {
-		*saddr = *(struct sockaddr_in *) info->ai_addr;
+		memcpy(saddr, info->ai_addr, sizeof(*saddr));
```

Comment:
This is just a fix to a weird alignment warning. I think this version looks nicer too.